config/policy-controller/kustomization.yaml

```diff
@@ -1,8 +1,21 @@
-resources:
-- manager.yaml
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kuadrant-
+
 images:
 - name: policy-controller
   newName: quay.io/kuadrant/policy-controller
   newTag: main
+
+
+resources:
+- ./namespace.yaml
+- ./manager.yaml
+- ../crd
+- ./rbac
+
+namespace: kuadrant-policy-controller
```

Comment:
Personally i think you would be better setting this up like it would be if it was it's own controller, and have the expected config structure for policy-controller setup in here (config/policy-controller). When you come to move it to it's own repo its just a matter of moving directory as is.

```
config/policy-controller
├── crd
│   ├── bases
│   │   ├── kuadrant.io_dnshealthcheckprobes.yaml
│   │   ├── kuadrant.io_dnspolicies.yaml
│   │   ├── kuadrant.io_dnsrecords.yaml
│   │   ├── kuadrant.io_managedzones.yaml
│   │   └── kuadrant.io_tlspolicies.yaml
│   ├── kustomization.yaml
│   ├── kustomizeconfig.yaml
│   └── patches
│       └── policy-patch.yaml
├── default
│   ├── issuer.yaml
│   ├── kustomization.yaml
│   └── namespace.yaml
├── deploy
│   └── local
│       ├── kustomization.yaml
│       └── manager_config_patch.yaml
├── kustomization.yaml
├── manager
│   ├── kustomization.yaml
│   └── manager.yaml
├── manager.yaml
└── rbac
    ├── dnsrecord_editor_role.yaml
    ├── dnsrecord_viewer_role.yaml
    ├── kustomization.yaml
    ├── leader_election_role_binding.yaml
    ├── leader_election_role.yaml
    ├── managedzone_editor_role.yaml
    ├── managedzone_viewer_role.yaml
    ├── role_binding.yaml
    ├── role.yaml
    ├── service_account.yaml
    ├── tlspolicy_editor_role.yaml
    └── tlspolicy_viewer_role.yaml
```

Does it not complain about `../crd` being outside the directory?